test/tests/time_steppers/timesequence_stepper/timesequence.i

```diff
@@ -0,0 +1,77 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = -1
+  xmax =  1
+  ymin = -1
+  ymax =  1
+  nx = 2
+  ny = 2
+  elem_type = QUAD9
+[]
+
+[Functions]
+  [./exact_fn]
+    type = ParsedFunction
+    value = t*t*(x*x+y*y)
+  [../]
+
+  [./forcing_fn]
+    type = ParsedFunction
+    value = 2*t*(x*x+y*y)-4*t*t
+  [../]
+[]
+
+[Variables]
+  [./u]
+    family = LAGRANGE
+    order = SECOND
+  [../]
+[]
+
+[ICs]
+  [./u_var]
+    type = FunctionIC
+    variable = u
+    function = exact_fn
+  [../]
+[]
+
+[Kernels]
+  [./td]
+    type = TimeDerivative
+    variable = u
+  [../]
+
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+
+  [./ffn]
+    type = UserForcingFunction
+    variable = u
+    function = forcing_fn
+  [../]
+[]
+
+[BCs]
+  [./all]
+    type = FunctionDirichletBC
+    variable = u
+    boundary = 'left right top bottom'
+    function = exact_fn
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  [./TimeStepper]
+    type = TimeSequenceStepper
+    time_knots  = '0   0.85 2 4'
```

Comment:
I'm ok with "knots" even though several others seem confused. Since you renamed the class, do you want to rename the parameter as well?